test/azlinux_test.go

```diff
@@ -111,6 +112,23 @@ func TestAzlinux3(t *testing.T) {
 	})
 }
 
+func signRepoAzLinux(gpgKey llb.State) llb.StateOption {
+	// key should be a state that has a public key under /public.key
+	return func(in llb.State) llb.State {
+		return in.Run(
+			dalec.ShArgs("gpg --import < /tmp/gpg/PUBLIC-RPM-GPG-KEY"),
```

Comment:
This path isn't correct anymore, it should be under `/tmp/gpg/public.key` in the `gpgKey` state. I think the reason it's still working though is because the public key doesn't actually need to be imported successfully to do the signing, so can we remove this line?

Reply:
Hah, yeah the 1st line is not doing anything at all since the second `dalec.ShArgs` is replacing it.

Reply:
Fixed.